test/test_IPy.py

```diff
@@ -68,6 +68,11 @@ class parseAddress(unittest.TestCase):
 
     # TODO: check for more invalid input
 
+    def testSpaces(self):
+        """Test if there is space(s) in inputted string."""
+        with self.assertRaises(ValueError):
```

Comment:
I saw that but I do not know what is the cause. I am using both 2.7 in 3.5 in local machine and it is fine :/.